src/mgmt/rpc.h

```diff
@@ -64,7 +64,10 @@
 		&ds_mgmt_hdlr_tgt_map_update_co_ops),			\
 	X(MGMT_TGT_MARK,						\
 		0, &CQF_mgmt_mark,					\
-		ds_mgmt_tgt_mark_hdlr, NULL)
+		ds_mgmt_tgt_mark_hdlr, NULL),				\
+	X(MGMT_TGT_SHARD_DESTROY,					\
+		0, &CQF_mgmt_tgt_shard_destroy,				\
+		ds_mgmt_hdlr_tgt_shard_destroy, NULL)
```

Comment:
By the way, do we need to bump DAOS_MGMT_VERSION above for this change?

Reply:
right

Reply:
hmm, do we need to add support for proto query like we did with object and pool RPCs?

Reply:
Currently, it seems unnecessary because we do not support interoperation among servers.